packages/shared/src/features/onboarding/hooks/useFunnelNavigation.ts

```diff
@@ -174,7 +175,7 @@ export const useFunnelNavigation = ({
     },
     // only run on mount
     // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
🙈

Reply:
I've updated the comment since now is not only on mount!

Reply:
If that's the case, we can then maybe remove the `eslint-disable` which I think they are referring to 👀